euphonic/qpoint_phonon_modes.py

```diff
@@ -149,7 +150,7 @@ def reorder_frequencies(self,
                 dot_mat = np.reshape(dots, (n_branches, n_branches))
 
                 # Find greatest dot product
-                for j in range(n_branches):
+                for _ in repeat(None, n_branches):
```

Comment:
Why do you consider `repeat(None, n_branches)` clearer than `range(n_branches)`?

Reply:
It's a little bit more explicit (that the outer loop does nothing but repeat), and goes straight to an iterator rather than via the iterable lazy object thing.

But this is very marginal and debatable; I'm just playing around really 😅 . Can go back to `range` if you think it would confuse/slow future maintenance.

The important part here was making it clear that `j` is not used, and `_` achieves that.

Reply:
Fixed. In the process, realised one of these loops was broken anyway 😆